### DISCUSSION TITLE
Problem with text line space in table

### DISCUSSION THREAD

**Discussion by QiangF** [kjambunathan/org-mode-ox-odt#119] [Q&A]
I find I can't change the line-height in "OrgTableContents" in the odt_styles_file, the exporter always falls to the default paragraph line spacing, which I want it to be "150%".

    <style:style style:name="OrgTableContents" style:family="paragraph" style:parent-style-name="Standard">
    <style:paragraph-properties fo:line-height="100%"
                                fo:padding="0cm"
                                fo:border="none">
    </style:paragraph-properties>
    </style:style>

**Comment by QiangF** [comment]
The problem with "150%" line spacing is that the text is not vertically centered in the table cell. It's probably a bug in libreoffice to add line space between text and table cell border.
![img_2021_10_31__10_13_22](https://user-images.githubusercontent.com/8192731/139563972-3fc1c546-0f36-4268-bf04-ecda0fbc29a2.png)

**Reply by kjambunathan** [reply]
> The problem with "150%" line spacing is that the text is not vertically centered in the table cell. It's probably a bug in libreoffice to add line space between text and table cell border.

Intuitively, yes that is what I think.  I noticed it while working on removing the "space" in this query.   Apart from padding etc. one of the contributors was the line spacing.

**Comment by QiangF** [comment]
In China, paragraph by default has a first line indentation,  and that is set by the default style

    <style:default-style style:family="paragraph">

Since indentation is only used in paragraphs with only text, if the default paragraph is controlled by the above default style, that style should not be used as the base by other styles, because most other styles do not have indentation.

I guess the default table cell paragraph style is hard coded in ox-odt.el. If so, maybe change it to depend on "Standard".

**Reply by kjambunathan** [reply]
> In China, paragraph by default has a first line indentation, and that is set by the default style
> 
> ```
> <style:default-style style:family="paragraph">
> ```
> 
> Since indentation is only used in paragraphs with only text, if the default paragraph is controlled by the above default style, that style should not be used as the base by other styles, because most other styles do not have indentation.
> 
> I guess the default table cell paragraph style is hard coded in ox-odt.el. If so, maybe change it to depend on "Standard".

This was a bug I introduced yesterday.  This should be fixed now.
